stl/inc/variant

```diff
@@ -1713,6 +1746,12 @@ _NODISCARD constexpr bool operator<=(monostate, monostate) noexcept {
 _NODISCARD constexpr bool operator>=(monostate, monostate) noexcept {
     return true;
 }
+#else // ^^^ _HAS_CXX20 vvv

```

Comment:
We have a couple of competing conventions for `#else` comments, this is similar to our `// ^^^ condition above / condition below vvvv` style but stops a bit short:
```suggestion
#else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
```

Reply:
On second thought, the overall form if this conditional is "if !condition then things else other things". We usually try to avoid negative conditions when there's an else. Would you mind flipping this into `#if _HAS_CXX20 #else #endif` form?

Reply:
Happy to :). I was just trying to follow the line of something I was elsewhere